themes/bootstrap3/js/facets.js

```diff
@@ -164,7 +164,9 @@ VuFind.register('sideFacets', function SideFacets() {
     // if the data element is undefined, we assume we are handling a click.
     var href = typeof data === "undefined" || typeof data.node.data.url === "undefined"
       ? $(this).attr('href') : data.node.data.url;
-    window.location.assign(href);
+    if (typeof href !== 'undefined') {
```

Comment:
Is this a generic fix that we should commit separately?

Reply:
No, it's only needed now when there can be an element without an href. That wasn't possible (at least with the default templates) before.